qiskit/synthesis/discrete_basis/generate_basis_approximations.py

```diff
@@ -121,15 +121,15 @@ def _process_node(node: Node, basis: list[str], sequences: list[GateSequence]):
 def generate_basic_approximations(
     basis_gates: list[str | Gate], depth: int, filename: str | None = None
 ) -> list[GateSequence]:
-    """Generates a list of ``GateSequence``s with the gates in ``basic_gates``.
+    """Generates a list of :class:`GateSequence`s with the gates in ``basis_gates``.
```

Comment:
```suggestion
    """Generates a list of :class:`GateSequence`\\ s with the gates in ``basis_gates``.
```

But I'm not dead certain that `GateSequence` is publicly documented.